ansys/dpf/core/dpf_operator.py

```diff
@@ -272,34 +291,37 @@ def __del__(self):
             pass
 
     def __str__(self):
-        """describe the entity
+        """Describe the entity.
         
         Returns
         -------
-        description : str
+        str
+            Description of the entity.
         """
         from ansys.dpf.core.core import _description
         return _description(self._message, self._server)
 
     def run(self):
-        """Evaluate this operator"""
+        """Evaluate the operator."""
```

Comment:
```suggestion
        """Evaluate this operator."""
```
I think this is better.